src/main/scala/com/scalaSparkSkillValidation/models/Customer.scala

```diff
@@ -0,0 +1,8 @@
+package com.scalaSparkSkillValidation.models
+import java.sql._
+
```

Comment:
Ok, please add ScalaDoc in any Class, Object, Methods - public mostly - I wont repeat that ask.

Reply:
Understood I'll add ScallaDoc to all project files

Reply:
ScalaDoc implemented